packages/gestalt/src/contexts/ExperimentProvider.js

```diff
@@ -0,0 +1,24 @@
+// @flow strict
+import { type Context, createContext, useContext } from 'react';
+
+type Experiment = {|
+  anyEnabled: boolean,
+  group: string,
+|};
+
+type ExperimentContextType = {|
+  [experimentName: string]: Experiment,
+|};
+
+const ExperimentContext: Context<ExperimentContextType> = createContext<ExperimentContextType>({});
+
+/**
+ * *ALPHA - DO NOT USE YET - MAY HAVE BREAKING CHANGES IN THE NEAR FUTURE*
+ */
+const ExperimentProvider = ExperimentContext.Provider;
+export default ExperimentProvider;
```

Comment:
For context, both Color and LinkNavigation providers, follow the pattern of custom-named props. If we export directly the prop is just value, which is the default provider prop.

https://github.com/pinterest/gestalt/pull/1996/files#diff-a7d0620f7dbfed6bca89e822ad18205902f10845a7039f4fa28028b464947faa

https://github.com/pinterest/gestalt/pull/1996/files#diff-a7d0620f7dbfed6bca89e822ad18205902f10845a7039f4fa28028b464947faa

Any thoughts?

Reply:
Not concerned about this implementation per se, just the consistency of implementations.

Reply:
I think OnLinkNavigationProvider probably could've used the same pattern of exporting the Provider directly, since onNavigation is passed directly through to `value`. ColorSchemeProvider is different, since it contains additional logic and markup. Unless I'm misunderstanding what you're asking about?

Reply:
You're right that it's not strictly necessary;

I was just referring to the pattern and the benefit of having autoGenerated prop tables for free based on the props.